src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Select.java

```diff
@@ -107,8 +110,9 @@ public void addField(Field field) {
 		if (field == null ) {
```

Comment:
Can you please replace all the tab characters with 4 spaces each throughout this file before you push the change?

Reply:
Sure. We can also do this for all other files once so we don't need to add comment for this in each PR.